### ISSUE TITLE
Mecanisme to report mistaken content

### ISSUE THREAD

**Issue by raphodn** [openfoodfacts/open-prices-frontend#98]
### Story

As a contributor, it may happen that a price is added with an error (wrong barcode, mistake when typing the price or choosing the location).

If we detect it, how can we report it ?

Alternatively, we could allow edition of your own content ?

**Comment by teolemon** [comment]
From my big report: 
![image](https://github.com/openfoodfacts/open-prices-frontend/assets/1689815/911c7833-840e-42d9-bc6c-61605255888b)
I scanned a cosmetic product. He showed me some chocolate chip muffins. I wish I could report that the product in Open Food Facts is incorrect.

**Comment by teolemon** [comment]
There is also the case where the same barcode represents several products. This is very common in the case of Aldi. It would be nice to have a button to be able to report duplicate products. This is not necessarily the role of Open price but it is a good time to realize the error. While waiting for the Nutri-Patrol backend to support this, we could write a key/value to the Folksonomy engine

**Comment by fabi003** [comment]
Good idea! Added a product with a wrong price date, and could not edit or delete it.

**Comment by raphodn** [comment]
We'll go with the "edit or delete your own data" approach, but we need new endpoints in the backend first : https://github.com/openfoodfacts/open-prices/issues/166

**Comment by raphodn** [comment]
@fabi003 it's now possible to delete prices ! #291

**Comment by fabi003** [comment]
Thanks you! Tried it and it works without problems.

**Comment by teolemon** [comment]
- #448

**Comment by TTalex** [comment]
Closing, fixed with #1824
